xbmc/video/VideoDatabase.cpp

```diff
@@ -4905,11 +4909,25 @@ void CVideoDatabase::UpdateTables(int iVersion)
     m_pDS->exec("ALTER TABLE settings ADD VideoStream integer");
     m_pDS->exec("ALTER TABLE streamdetails ADD strVideoLanguage text");
   }
+
+  if (iVersion < 104)
+  {
+    m_pDS->exec("ALTER TABLE tvshow ADD duration INTEGER");
+
+    std::string sql = PrepareSQL("SELECT idShow, MAX(c%02d) FROM episode GROUP BY idShow", VIDEODB_ID_EPISODE_RUNTIME);
```

Comment:
I feel like showing a too long runtime is better than showing a too short runtime.
Like I have to go somewehere in 30mins, I would probably like to watch something  shorter than 30 min.
Just for the mindset.